reminders/scheduler.py

```diff
@@ -109,6 +110,12 @@ def _get_function(method_type):
         if method_type in (ReminderType.WEBSITE, ReminderType.WEBSITE_EMAIL):
             function_to_invoke.append(create_notification)
 
+        if method_type in ReminderType.RUN_ALGORITHM:
```

Comment:
I think because ReminderType.RUN_ALGORITHM is not tuple you need to use "==" instead of "in"
```suggestion
        if method_type == ReminderType.RUN_ALGORITHM:
```

Reply:
Done, thanks 😄